src/journal_frontend/src/App.vue

```diff
@@ -0,0 +1,71 @@
+<template>
```

Comment:
I haven't seen vue code before. This is quite elegant. I'll read up on it before I can provide any useful review. :-)